File: 2.Traverse/Javascript/5.BlueRedBoard.js
function lastPaintedColor(board) {
    for (let row of board) {
        if (row === "RRRRRRRR") {
            return 'R';
        }
    }
    return 'B';
}

// Hardcoded single board input
const board = [
    "....B....",
    "....B....",
    "....B....",
    "RRRRRRRR",
    "....B....",
    "....B....",
    "....B....",
    "....B....",

];

console.log(lastPaintedColor(board));

File: 2.Traverse/Python/5.BlueRedBoard.py
def last_painted_color(board):
    for row in board:
        if row == "RRRRRRRR":
            return 'R'
    return 'B'

# Hardcoded single board input
board = [
    "BBBBBBBB",
    "BBBBBBBB",
    "BBBBBBBB",
    "BBBBBBBB",
    "BBBBBBBB",
    "BBBBBBBB",
    "BBBBBBBB",
    "RRRRRRRR"
]

print(last_painted_color(board))
